SabPicariello/io-app-test
# Added Switch StatoCancellazioneProfilo

# This PR Depends on #1 
## Short description
Added a `RemoteSwitch` that indicates the profile deletion status.

## List of changes proposed in this pull request
- locales/en/index.yml: added a label for the profile deletion status.
- locales/it/index.yml: added a label for the profile deletion status.
- ts/components/PreferencesListItem.tsx: added support to edit the style of the component and exported its properties.
- ts/screens/profile/ProfileTest.tsx: added a `PreferencesListItem` to show the profile deletion status in new profile page through a `RemoteSwitch`.
## How to test

Set the environment to dev, run the application and open the new profile page. Then:
1) Verify that the new switch is not clickable and shows a loading spinner while loading.
2) Make a POST request to dev server:
     - url : `/user-data-processing`
     - body : `{"choice":"DELETE"}`
Reload the new profile page by swiping down and verify that the switch for the profile deletion status is active.
3) Make a DELETE request to dev server:
     - url : `/user-data-processing/DELETE`
Reload the new profile page by swiping down and verify that the switch for the profile deletion status is not active.
